labels.py

```diff
@@ -0,0 +1,26 @@
+from cProfile import label
+from lib2to3.pytree import LeafPattern
+from Labelling.config.constants import TEST_IMG_DIR_PATH
+
+import shutil
+import os
+
+
+LABEL_DIR = TEST_IMG_DIR_PATH
+NEW_PATH = os.path.abspath('Labels')
+
+print(NEW_PATH)
+
+for path, dirs, files in os.walk(LABEL_DIR):
+    for file in files:
+        if file.endswith('json'):
+            OLD_LOC = os.path.join(path, file)
+            OLD_FOLDER = (os.path.dirname(OLD_LOC).split('\\')[-1])
```

Comment:
I think there is an [os command to do this split](https://docs.python.org/3/library/os.path.html#os.path.split)

Reply:
This will important for non-windows users.